scripts/ssm-build.sh

```diff
@@ -197,3 +200,16 @@ fpm -s pleaserun -t "$PACK_TYPE" \
 --depends apel-ssm \
 --package "$BUILD_DIR" \
 /usr/bin/ssmreceive
+
+if [ "$OS_EXTENSION" == "_all" ]
+then
+    # Check the resultant debs for 'lint'
+    echo "Possible Issues to Fix:"
```

Comment:
We could save on repetition by putting the two `echo`s outside of the if-then block.

Reply:
resolved in next commit